packages/dcos-integration-test/extra/test_cockroach_config.py

```diff
@@ -0,0 +1,28 @@
+"""
+Testing the CockroachDB setup in DC/OS.
+"""
+import logging
+import subprocess
+from time import sleep
+
+
+__maintainer__ = 'tweidner'
+__contact__ = 'security-team@mesosphere.io'
+
+
+LOG = logging.getLogger(__name__)
+
+
+def test_config_change_applied():
+    subprocess.check_call(
+        ['sudo', 'systemctl', 'restart', 'dcos-cockroachdb-config-change'],
+    )
+    # Wait a few moments for the CRDB config change script to finish.
+    sleep(3)
+    result = subprocess.check_output(
+        ['sudo', 'journalctl', '-u', 'dcos-cockroachdb-config-change', '--no-pager'],
+        universal_newlines=True,
+    )
+    # Indicator for successful config update via the CockroachDB SQL client.
+    expected_output = 'CONFIGURE ZONE 1'
+    assert expected_output in result
```

Comment:
polling here for a few seconds may be a better way to test this.

Reply:
It definitely is. I find myself writing this polling code over and over again.